.github/workflows/black.yml

```diff
@@ -39,4 +36,4 @@ jobs:
       - name: Push changes
         uses: stefanzweifel/git-auto-commit-action@v4
         with:
-          commit_message: Apply formatting changes
+          commit_message: Apply formatting changes
```

Comment:
Can you update this message to mention black explicitly. Part of the reason it took so long to catch this was I saw "formatting changes" and thought it was doing both in one step. The same goes for the `isort` step.